test/test.bats

```diff
@@ -21,7 +21,7 @@ bats_load_library bats-file
 
 @test "trivy config sarif report" {
   # trivy config --format sarif --output  config-sarif.test .
-  run ./entrypoint.sh '-a config' '-b sarif' '-h config-sarif.test' '-j .'
+  run ./entrypoint.sh '-a config' '-b sarif' '-h config-sarif.test' '-j .' '-q ./test/data/main.tf'
```

Comment:
I'm unclear why this test was changed. This would only scan the terraform `main.tf` that you've supplied. Was that the intent?

Reply:
Hi simar, I think this was a mistake. I hadn't realised I'd forgotten to remove `'-q ./test/data/main.tf'`  - Good catch thanks